ui/components/MesheryAdapterPlayComponent.js

```diff
@@ -237,8 +242,28 @@ class MesheryAdapterPlayComponent extends React.Component {
       }
       )
       .catch(err =>
-        console.log("error at addon fetch: "+err)
+        console.log("error at addon fetch: " + err)
       )
+    fetchAvailableNamespaces()
+      .then(res => {
+        let namespaces = []
+        res?.namespaces?.map(ns => {
+          namespaces.push(
+            {
+              value: ns?.namespace,
+              label: ns?.namespace
+            }
+          )
+        })
+        if (namespaces.length === 0) {
+          namespaces.push({
+            value: "default",
+            label: "default"
+          })
+        }
+        self.setState({ namespaceList: namespaces })
+      })
+      .catch(err => console.log(err))
```

Comment:
Is it intentional or should it be changed to the `Snackbar Error`?

Reply:
@vineethvanga18 thoughts? 

If this errors out, it could mean that MeshSync data isn't being fed into the Meshery database. 

@utkarsh-pro, @kumarabd how do we hook into the notification system with probable cause and suggested remediation with errors initiated from the UI?

Reply:
@vineethvanga18 what do you suggest we do here?

Reply:
@leecalcote At other places where we used graphql(operator status, addons), we just logged the error to console.

Reply:
If we encounter an error it will be due to meshsync failing to collect that data
we may consider coming up with specific error codes to these

Reply:
Thank you.